### ISSUE TITLE
What are the external hardware components of the DJI remote control? How to use ADC2PPM?

### ISSUE THREAD

**Issue by kalading2** [EvgenyDD/ADC2PPM#1]

**Comment by EvgenyDD** [comment]
The hardware is just simple. Pins 5,6,7,8 (4 ADC inputs) are soldered to the joystick central pins of the DJI RC. Pin 1 is PPM output and connected to "USB RC simulator dongle" which can be bought from eBay.

**Comment by kalading2** [comment]
Does dji RC /dji RC Pro remote control work？

**Comment by EvgenyDD** [comment]
Yes, it should be turned on for powering potentiometers inside the joysticks, so ADC in STM32 will get analog signals from the potentiometers. Note that disassembling the DJI RC and soldering to the PCB may void warranty.

**Comment by kalading2** [comment]
@EvgenyDD  ,thank you for your help.
